TicketsDemo/App_Start/NinjectWebCommon.cs

```diff
@@ -65,19 +66,31 @@ private static IKernel CreateKernel()
         private static void RegisterServices(IKernel kernel)
         {
             kernel.Bind<ITicketRepository>().To<TicketRepository>();
-            kernel.Bind<ITrainRepository>().To<TrainRepository>();
+            //kernel.Bind<ITrainRepository>().To<TrainRepository>();
+            kernel.Bind<ITrainRepository>().To<TrainRepositoryCSV>();
```

Comment:
I don't see TrainRepositoryCSV anywhere in the PR. Was it committed/pushed to github? Or you've just utilized the one, already developed by me many years ago and just lurked inside the TicketsDemo.CSV directory? That's not the technical solution :)